samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/shared/impl/assessment/SecureDeliveryServiceImpl.java

```diff
@@ -186,7 +209,7 @@ public PhaseStatus validatePhase(String moduleId, Phase phase, PublishedAssessme
 		
 		SecureDeliveryModuleIfc module = secureDeliveryModules.get( moduleId );
 		
-		if ( moduleId == null || NONE_ID.equals( moduleId ) || module == null || !module.isEnabled() )
+		if ( moduleId == null || NONE_ID.equals( moduleId ) || module == null || !module.isEnabled(assessment.getPublishedAssessmentId()) )
```

Comment:
Maybe use stringutils.isblank on moduleId?

Reply:
really don't want to refactor samigo code if not needed